### ISSUE TITLE
Port Extensions: Add sagas to support VotingReputation

### ISSUE THREAD

**Issue by danbr** [JoinColony/colonyCDapp#129]
One thing to note is that this doesn't resolve #113 entirely as we'll still be missing sagas specific to VotingReputation, as well as the functionality of enabling this extension (and possibly more).

Could you open a separate issue to "finish" porting VotingReputation and mark it as blocked by #112 / #121?

_Originally posted by @jakubcolony in https://github.com/JoinColony/colonyCDapp/pull/123#pullrequestreview-1209967290_

**Comment by Julianahlk** [comment]
Closing this issue because @jakubcolony will be implementing everything mentioned in this issue as part of #130.
